_posts/2016-02-25-usage-tests.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: post
+title: Usage Tests
+---
+
+Tests suites have a number of purposes.
+Perhaps most commonly you think of the test suite as validating the behavior of your code.
+Absolutely.
+But they also serve as examples of using the lib.
+Let's make that more explicit.
```

Comment:
I think the intro paragraph could use more of a hook—really grab me with why I want to keep reading this post. Maybe even just something as simple as "you might not realize they can serve as examples of using the lib." Or "are you taking advantage of the examples lib authors have given? Are you making such examples?" etc etc